apps/omg_watcher/lib/omg_watcher/api/in_flight_exit.ex

```diff
@@ -106,6 +106,9 @@ defmodule OMG.Watcher.API.InFlightExit do
 
       {:error, :utxo_not_found} ->
         {:halt, {:error, :tx_for_input_not_found}}
+
+      {:error, error} = e when is_atom(error) ->
```

Comment:
instead of a catch-all-errors clause, please pattern match the expected atom (`:no_deposit_for_given_blknum` I presume).

I would also advocate translating this particular error into something more meaningful like (`deposit_input_spent_ife_unsupported`)